tasks/src/main/scala/dagr/tasks/DagrDef.scala

```diff
@@ -65,4 +67,6 @@ object DagrDef {
   /** A String that represents the prefix or basename of a filename. */
   type FilenamePrefix = String
 
+  /** Implicit that will convert an Option[Dependable] to a Dependable when needed. */
+  implicit def optionDependableToDependable(maybe: Option[Dependable]): Dependable = EmptyDependable.optionDependableToDependable(maybe)
```

Comment:
Yeah, I felt like the implementation should live with all the Dependable stuff, but I wanted it to be imported into scope through `DagrDef`.